com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Utility/HDUtils.cs

```diff
@@ -142,33 +142,46 @@ internal static float ProjectionMatrixAspect(in Matrix4x4 matrix)
 
         internal static Matrix4x4 ComputePixelCoordToWorldSpaceViewDirectionMatrix(float verticalFoV, Vector2 lensShift, Vector4 screenSize, Matrix4x4 worldToViewMatrix, bool renderToCubemap, float aspectRatio = -1)
         {
+            Matrix4x4 viewSpaceRasterTransform;
             aspectRatio = aspectRatio < 0 ? screenSize.x * screenSize.w : aspectRatio;
 
-            // Compose the view space version first.
-            // V = -(X, Y, Z), s.t. Z = 1,
-            // X = (2x / resX - 1) * tan(vFoV / 2) * ar = x * [(2 / resX) * tan(vFoV / 2) * ar] + [-tan(vFoV / 2) * ar] = x * [-m00] + [-m20]
-            // Y = (2y / resY - 1) * tan(vFoV / 2)      = y * [(2 / resY) * tan(vFoV / 2)]      + [-tan(vFoV / 2)]      = y * [-m11] + [-m21]
+            if (verticalFoV < 0)
```

Comment:
@adrien-de-tocqueville  could you add an explicit argument instead of hacking with verticalFoV ? :)

i.e ComputePixelCoordToWorldSpaceViewDirectionMatrix(.... , bool isOrthographic = false).

and be a bit more versose on the comment:
// For ortho cameras, project the skybox with no perspective
(maybe add fobugzz link, thanks)

Reply:
I replicated the aspectRatio hack haha, but ok

Reply:
oh haven't noticed it :).  but let's be cleaner